ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/handlers/JsonRpcExecutorHandler.java

```diff
@@ -34,6 +34,9 @@
 public class JsonRpcExecutorHandler {
   private static final Logger LOG = LoggerFactory.getLogger(JsonRpcExecutorHandler.class);
 
+  // Default timeout for RPC calls in seconds
+  private static final long DEFAULT_TIMEOUT_MILLISECONDS = 30000;
```

Comment:
30 seconds is a very long time. Is that time that long on purpose?

Reply:
30 seconds looks good for me. API and HTTP timeouts are in general around 60 seconds.
We can start with this configuration and improve it in the future if needed.